### PR TITLE
Revamp the AORs page of SRE

### PR DESCRIPTION
**Pull request by spirosoik**
#### Summary
As a team we defined our purpose, goals/actions and our principles. In parallel in the same page we define how we work and how you can interact with us.

### CODE REVIEW

**Review by stafot — DISMISSED** [review]

**Review by jwilander — DISMISSED** [review]

File: operations/research-and-development/organization/sre.md [hunk 1]
```diff
@@ -1,22 +1,58 @@
...
-  - SLOs for internal/external services
-  - Migration from self to cloud
-  - Workspaces management (deletion, export)
-  - Systems and processes automation
-  - Testing environments for engineering (spinwicks)
-- Cloud On-call
-  - Incident management
-  - Support teams adopt Cloud practices
+### Who We Are
+Site Reliability Engineering (SRE) team ensures the availability of Mattermost Cloud user-facing services, building the tools and automation to monitor and enable this availability. These user-facing services include multiple environments including testing, development and production, among others.
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md:4]:
```suggestion
The Site Reliability Engineering (SRE) team ensures the availability of Mattermost Cloud user-facing services by building the tools and automation to monitor multiple environments, including testing, development, and production, among others.
```

File: operations/research-and-development/organization/sre.md [hunk 2]
```diff
@@ -1,22 +1,58 @@
...
+
+### Vision
+Enable developers to build reliable, scalable, cost efficient services having in mind the Mattermost Cloud customer's reliability targets applying SRE principles (observability, SLOs etc.).
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
Enable developers to build reliable, scalable, cost efficient services that keepthe Mattermost Cloud customer's reliability targets and SRE principles (observability, SLOs etc.) in mind.
```

File: operations/research-and-development/organization/sre.md [hunk 3]
```diff
@@ -1,22 +1,58 @@
...
+
+### Actions
+- On-call, Incident Management and Incident Review
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md:10]:
```suggestion
- On-call, Incident Management, and Incident Review
```

**Review by stylianosrigas — DISMISSED** [review]
Great stuff 🚀

File: operations/research-and-development/organization/sre.md [hunk 4]
```diff
@@ -1,22 +1,58 @@
...
+- Change management
+- Influence and encourage best practices
+- Keep documentation, training and Runbooks fresh
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
- Maintain and update documentation, training, and Runbooks
```

File: operations/research-and-development/organization/sre.md [hunk 5]
```diff
@@ -1,22 +1,58 @@
...
+- Empower developers with self-served capabilities
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
- Empower developers with self-serve tools
```

File: operations/research-and-development/organization/sre.md [hunk 6]
```diff
@@ -1,22 +1,58 @@
...
+- Meet & review reliability, quality and cost KPIs regularly
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
- Meet & review reliability, quality, and cost KPIs regularly
```

File: operations/research-and-development/organization/sre.md [hunk 7]
```diff
@@ -1,22 +1,58 @@
...
+### Principles
+- Be proactive instead of reactive
+- Be open and data driven
+- Embrace risk and chaos
+- Promote and embrace communication
+- Evangelise cost effectiveness
+- Evangelise and adopt SRE & DevOps practices 
+
+### Ask for Help
+If you need our assistance please use the following follow the [General Workflow](#general-workflow)
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
If you need our assistance please follow the [General Workflow](#general-workflow)
```

File: operations/research-and-development/organization/sre.md [hunk 8]
```diff
@@ -1,22 +1,58 @@
...
+
+We can also be reached out:
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
You can also reach us on the Mattermost Community Server as follows:
```

File: operations/research-and-development/organization/sre.md [hunk 9]
```diff
@@ -1,22 +1,58 @@
...
+1. For support questions related to production you can reach us in   `~cloud-support ` channel.
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md:29]: Consider adding direct links to these two channels for easy access

File: operations/research-and-development/organization/sre.md [hunk 10]
```diff
@@ -1,22 +1,58 @@
...
+2. For all the other questions you can reach us in `~cloud-sre-team` channel
+
+### How we work
+We maintain every quarter our OKRs as epics in this [Board](https://mattermost.atlassian.net/jira/software/c/projects/CLD/boards/109/roadmap?statuses=2%2C4) which are divided in the following areas:
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
Each quarter, we maintain our OKRs as epics in this [board](https://mattermost.atlassian.net/jira/software/c/projects/CLD/boards/109/roadmap?statuses=2%2C4) which is divided into the following areas:
```

File: operations/research-and-development/organization/sre.md [hunk 11]
```diff
@@ -1,22 +1,58 @@
...
+- Reliability & Resiliency
+- Availability & Cost Optimisation
+- Automation
+- Security
+- Documentation
+- Keep the lights on
+
+#### General Workflow
+1. Mattermost members open new issues in [SRE Board](https://mattermost.atlassian.net/jira/software/c/projects/CLD/boards/109)
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
1. Mattermost members open new issues in the [SRE Board](https://mattermost.atlassian.net/jira/software/c/projects/CLD/boards/109)
```

File: operations/research-and-development/organization/sre.md [hunk 12]
```diff
@@ -1,22 +1,58 @@
...
+2. Every new issues goes through an Issue Triage weekly
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
2. New issues are reviewed weekly via Issue Triage.
```

File: operations/research-and-development/organization/sre.md [hunk 13]
```diff
@@ -1,22 +1,58 @@
...
+
+### Areas of Ownership
+The team regularly works on the following tasks, in the order of priority:
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
The team regularly works on the following tasks, listed below in priority order:
```

File: operations/research-and-development/organization/sre.md [hunk 14]
```diff
@@ -1,22 +1,58 @@
...
+- Ensuring availability of Mattermost Cloud user-facing services
+- Ensuring availability of community.mattermost.com (eg. daily releases, calls)
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
- Ensuring availability of community.mattermost.com (e.g. daily releases, calls)
```

File: operations/research-and-development/organization/sre.md [hunk 15]
```diff
@@ -1,22 +1,58 @@
...
+- Ensuring the availability of the internal Cloud Platform (eg. self-served testing envs, Observability Platform, GitOps Platform, IaC platform )
```
Comment (cwarnermm) [inline operations/research-and-development/organization/sre.md]:
```suggestion
- Ensuring the availability of the internal Cloud Platform (e.g. self-serve testing environments, Observability Platform, GitOps Platform, IaC platform )
```

**Review by cwarnermm — COMMENTED** [review]
Really great update, @spirosoik! I've provided editorial feedback to consider. Keep what you agree with and feel free to dismiss what doesn't work well from your perspective.

**Comment by spirosoik** [comment]
@stylianosrigas @stafot @jwilander could please approve again?

**Review by stafot — APPROVED** [review]

**Review by angeloskyratzakos — APPROVED** [review]

**Review by stylianosrigas — APPROVED** [review]

**Pull request merged**